recipes-wpe/wpeframework/wpeframework-plugins_git.bb

```diff
@@ -6,7 +6,6 @@ PR = "r1"
 require include/wpeframework-plugins.inc
 
 SRC_URI = "git://github.com/WebPlatformForEmbedded/WPEFrameworkPlugins.git;protocol=git;branch=master \
-           file://0001-WebKitBrowser-Add-Time-to-dependencies.patch \
```

Comment:
webkit browser is getting into suspended state without this change

Reply:
That should be fixed on newer revisions of wpewebkit. Are you still seeing that with the latest?

Reply:
in our build we are using https://github.com/WebPlatformForEmbedded/meta-wpe/blob/morty/recipes-wpe/wpewebkit/wpewebkit_20170728.bb

Reply:
OK - I'll check it. I did not run into those issues on my tests yesterday. WPEWebKit should have a fix for time changes during start/run time.

Reply:
re-added it, saw the same and will debug this later. Want to make sure this PR goes through first.